app/dashboard/models.py

```diff
@@ -2788,7 +2788,7 @@ class Profile(SuperModel):
     last_validation_request = models.DateTimeField(blank=True, null=True, help_text=_("When the user requested a code for last time "))
     encoded_number = models.CharField(max_length=255, blank=True, help_text=_('Number with the user validate the account'))
     sybil_score = models.IntegerField(default=-1)
-
+    ignore_tribes = ArrayField(models.IntegerField(), blank=True, default=list)
```

Comment:
Why use an `ArrayField` here instead of a `ManyToManyField`? Will cause problems if a tribe is later removed and we aren't cascading the deletion down to the reference.

Reply:
I used the `ArrayField` because the spec in the issue requests for one object to store the ignored tribes. I got your point, at this moment doesn't matter if a tribe is deleted because all ids in this list will be excluded

Reply:
I used the `ArrayField` because the spec in the issue requests for one object to store the ignored tribes. I got your point, at this moment doesn't matter if a tribe is deleted because all ids in this list will be excluded

Reply:
Replaced with ManyToMany